### ISSUE TITLE
Delete through Plex instead of the filesystem to prevent: "File does not exist" 

### ISSUE THREAD

**Issue by wolph** [scottrobertson/plex-autodelete#2]
First of all, great script. Really looks like some awesome work :)

I do have a question though, how can I make it delete from remote servers? It just tells me "File does not exist" for every file right now. I thik it would be better to delete through the Plex API instead of through the filesystem.

**Comment by scottrobertson** [comment]
Yeah, I thought about after I made this, it would make much more sense to do it via the API. 

I don't really have time to work on that right now, but feel free to submit a PR if you can work it out.

**Comment by wolph** [comment]
I personally have no need anymore, the plex-delete script (https://github.com/WoLpH/plex-delete) does most of what I need already and was slightly easier for me to modify. I quite like Ruby but I'm more experienced with Python :)
